Content.Server/Construction/ConstructionSystem.Guided.cs

```diff
@@ -33,7 +38,8 @@ private void OnGuideRequested(RequestConstructionGuide msg, EntitySessionEventAr
 
         private void AddDeconstructVerb(EntityUid uid, ConstructionComponent component, GetOtherVerbsEvent args)
         {
-            if (!args.CanAccess)
+            if (!args.User.IsInSameOrParentContainer(args.Target) ||
+                !_sharedInteractionSystem.InRangeUnobstructed(args.User, args.Target, ignoreInsideBlocker: true))
```

Comment:
i think we can just set the ignoreinsideblocker to true for args.CanAccess in general tbh

Reply:
ok, did that.